static/src/js/components/OrderOptionList/OrderOptionList.jsx

```diff
@@ -208,6 +220,26 @@ const OrderOptionList = () => {
 
               return (
                 <>
+                  {
+                    (!providerIds || providerIds.length === 0)
+                      ? (
+                        <Alert variant="info" className="mb-4">
```

Comment:
I like this implementation of showing which providers we are showing, but this list page is different than all of the other list page. Don't really have a strong opinion, curious to know what everyone else thinks.

Reply:
Right and that's because this list page is different from all the others. Other list pages don't require a user to have providers. Anyone can see anyone else's records. This particular list is being rendered specifically from the providers a user has available and nothing more.